src/gc/unix/numa_gc.h

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __NUMA_H
+#define __NUMA_H
+
+#define PAGE_READWRITE                  0x04
+#define MEM_COMMIT                      0x1000
+
+#define ERROR_INVALID_HANDLE 6L
```

Comment:
I think I am going to change course here, I am in the middle of trying to bring the standalone GC up to feature parity with the the built in GC and want to avoid going on tangential projects where it's not necessary. I agree with your suggestion here to get rid of the windows emulation as much as possible, but looking at the GC code, numa is disabled for the PAL even on the built in GC. So my plan now is to keep the windows implementation with the API changes you mention below, and leave stubs for numa on Unix.

Reply:
> numa is disabled for the PAL even on the built in GC

? NUMA was enabled for PAL more than a year ago. See https://github.com/dotnet/coreclr/pull/10938

Reply:
The numa code is there, but unused from what I can tell. virtual_alloc_commit_for_heap is the only place I see numa referenced in the GC, and it's under `#if defined(MULTIPLE_HEAPS) && !defined(FEATURE_REDHAWK) && !defined(FEATURE_PAL) && !defined(BUILD_AS_STANDALONE)` for at least a few years back.